styles.css

```diff
@@ -132,6 +133,209 @@ body {
 }
 
 
+/* New code */
+    .grid-container {
+        display: grid;
+        grid-template-rows: repeat(4, auto);
+        grid-template-columns: 327px;
+        justify-content: center;
+        background-color: #E5E5E5;
+        padding: 114px 0;
+        row-gap: 88px;
+    }
+
+    .grid-item {
+        display: flex;
+        flex-direction: column;
+        border-radius: 16px;
+        border: 1px solid #dfe1e6;
+        background-color: #FFFFFF;
+    }
+
+    .grid-item img {
+        display: flex;
+        justify-content: center;
+        margin: 16px;
+    }
+
+    .grid-item h2 {
+        display: flex;
+        font-size: 32px;
+        line-height: 44px;
+        font-weight: 700;
+        margin: -3px 16px;
+    }
+
+    .details {
+        display: flex;
+        font-size: 13px;
+        line-height: 16px;
+        font-weight: 600;
+        margin: 0 16px;
+    }
+
+    .details ul {
+        margin: 0;
+        padding: 0;
+    }
+
+    .details li {
+        list-style-type: none;
+        display: flex;
+        align-items: center;
+        margin-top: 7px;
+    }
+
+    .details img {
+        margin: 16px 11px;
+    }
+
+    .worksText {
+        font-size: 15px;
+        line-height: 24px;
+        font-weight: 400;
+        margin: 8px 16px;
+    }
+
+    .worksTag {
+        font-size: 12px;
+        line-height: 16px;
+        font-weight: 500;
+    }
+
+    .worksTag ul {
+        margin: 2px 16px;
+        padding: 0;
+        display: flex;
+
+    }
+
+    .worksTag li {
+        list-style-type: none;
+        background-color:#EBEBFF;
+        padding: 4px 12px;
+        margin-right: 8px;
+        margin-top: 5px;
+        border-radius: 8px;
+        color: #6070FF;
+    }
+
+    .worksButton {
+        width: 124px;
+        height: 48px;
+        margin: 20px 16px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+
+    .aboutMe {
+        padding: 114px 24px;
+    }
```

Comment:
- [ ] Please follow the Figma design as a project requirement for making the top-right border-radius as shown in the Figma design for the about section.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219881419-25b1b8be-51a7-4744-a245-074b97132475.png)

### Figma Design 
![image](https://user-images.githubusercontent.com/68030297/219881439-c8e610a1-0aa1-40fa-9d88-7920ecc106cb.png)

- **NOTE ⚠️:** currently the about section background color inherits from the body element background color, so the border-radius for the about section won't show the result until you change the body background color.